webapp/src/index.tsx

```diff
@@ -74,9 +77,33 @@ export default class Plugin {
     public async initialize(registry: any, store: WebappStore) {
         setupRedux(registry, store);
 
+        const usersState = store.getState().entities.users;
+        const currentUser = usersState.profiles[usersState.currentUserId] || {};
+        const userLocale = currentUser.locale || 'en';
+
+        const translations: any = {
+            en: localeEn,
+            es: localeEs,
+        };
+
+        const withIntl = (Component: any) => {
```

Comment:
I think we can avoid this and register the translations with the webapp like playbooks does: https://github.com/mattermost/mattermost-plugin-playbooks/blob/be874bbd629b16379a13bafcb67ecdde4ff67608/webapp/src/index.tsx#L142

Reply:
Good point, I'll take a look.

Reply:
Applied, it is cool. It is tricky because you have to tell webpack to use the shared `ReactIntl` for that, if not it is not finding the `IntlProvider`. But now it is way simpler, for sure.